global.json

```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "5.0.302"
+    "version": "6.0.100-rc.2.21505.57"
```

Comment:
Use released dotnet version.

Reply:
I will fix it in the next PR, where i will update main template.